content/account_management/faq/how-do-i-configure-azure-ad-as-a-saml-idp.md

```diff
@@ -13,64 +13,75 @@ further_reading:
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure4.png" alt="Active_Directory_-_Microsoft_Azure4" responsive="true" popup="true">}}
+11. Set `user.mail` as **User Identifier** Value:
 
-10. Navigate to https://app.datadoghq.com/saml/saml_setup, find the Service Provider Entity ID & Assertion Consumer Service URL on the right hand of the page.  Copy and paste those values in the "Identifier" and "Reply URL" text forms respectively
-    {{< img src="account_management/faq/Step10Redo.png" alt="Step10Redo" responsive="true" popup="true">}}
-    {{< img src="account_management/faq/Step10Redo2.png" alt="Step10Redo2" responsive="true" popup="true">}}
+    {{< img src="account_management/faq/user_identifier.png" alt="User Identifier" responsive="true" popup="true" style="width:70%;">}}
 
-11. On the next page Download Metadata (XML), check the box confirming you have configured SSO and press next. Use this file in step 17
+12. Enter your **Notification Email** at the bottom of the page for when the active signing certificate approaches its expiration date, notifications are then sent to this email address with instructions on how to update the certificate:
 
-12. Enter in an email address at which you would like to be notified about maintenance issues
+    {{< img src="account_management/faq/notification_email.png" alt="Notification email" responsive="true" popup="true" style="width:70%;">}}
 
-13. Once you are back to the applications main page, navigate to "Attributes"
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure6.png" alt="Active_Directory_-_Microsoft_Azure6" responsive="true" popup="true">}}
+13. Click at the bottom of the page on Step 5 **Configure DatadogSSO_test**.
 
-14. In "SAML Token Attributes", hover over the line where "TYPE" is "user attribute (nameid)" (usually the first one) and click on the pencil icon to edit
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure7.png" alt="Active_Directory_-_Microsoft_Azure7" responsive="true" popup="true">}}
+14. Scroll down to the Step 3 of the *Configure DatadogSSO_test for single sign on* section, and download the **SAML XML Metadata** File.
 
-15. Change the attribute value to "user.mail" and press the check button
-    {{< img src="account_management/faq/Active_Directory_-_Microsoft_Azure14.png" alt="Active_Directory_-_Microsoft_Azure14" responsive="true" popup="true">}}
+15. Go at the top of your **SSO Configuration section** and click **Save**.
+
+16. Navigate back to [Datadog SAML page][1] and upload the **SAML XML Metadata** file downloaded in step 14:
+
+    {{< img src="account_management/faq/SAML_Configuration___Datadog10.png" alt="SAML_Configuration___Datadog10" responsive="true" popup="true" style="width:70%;">}}
+
+17. Make sure to press the **Upload File** button after having chosen the XML file to upload.
+
+18. And that’s it! It should now say *SAML is ready* and that Valid IdP metadata installed.
```

Comment:
18. And that’s it! It should now say *SAML is ready* and that Valid IdP metadata is installed.